CODEOWNERS

```diff
@@ -1,126 +1,142 @@
 # Search and storage and infra is the default owner
-* @getsentry/owners-snuba @getsentry/ops
```

Comment:
this entire file got reformatted, please revert that

Reply:
Not sure why but every time I try to save it, it reverts to reformatting it. Do you know how to fix that?

Reply:
turn off format-on-save in your editor

Reply:
These changes are being committed via `style(lint): Auto commit lint changes` commits (the bot? is committing them). So it seems I cannot remove those?